config/wrapper/pci_input.conf

```diff
@@ -1,23 +1,92 @@
 [io_nvme_fvt]
+devices = interfaces:0
 disk = disks:0
+lv_disks = disks:0
+htx_disks = disks:1
 disks = disks:all
-device = interfaces:0
 pci_device = functions:0
+pci_devices = functions:all
 module = driver
 
 [io_fc_fvt]
 disk = mpath_disks:0
-lv_disks = mpath_disks:1
-htx_disks = mpath_disks:2
+lv_disks = mpath_disks:0
+htx_disks = mpath_disks:1
 disks = mpath_disks:all
 wwids = mpath_wwids:all
 pci_device = functions:0
 pci_devices = functions:all
 module = driver
 
+[io_nvmf_rhel8_fvt]
+disk = mpath_disks:0
+lv_disks = mpath_disks:0
+htx_disks = mpath_disks:1
+disks = mpath_disks:all
+wwids = mpath_wwids:all
+pci_device = functions:0
+pci_devices = functions:all
+module = driver:all
+
+[io_nvmf_fvt]
+device = disks:0
+disk = disks:0
```

Comment:
device is a nvme device and disk is a nvme space.. hope that is taken care

Reply:
yes that is taken care via input file.